messaging.rst

```diff
@@ -85,71 +85,305 @@ Messages
+| channel_identifier    | uint256    | 1338                                                                 |
++-----------------------+------------+----------------------------------------------------------------------+
+| balance_hash          | bytes32    | 0x1d9479b298eb0a60edaf962f4cf092465456ad7a0265dfe28a0fe3a2a8ecef4e   |
++-----------------------+------------+----------------------------------------------------------------------+
+| nonce                 | uint256    | 1                                                                    |
++-----------------------+------------+----------------------------------------------------------------------+
+| additional_hash       | bytes32    | 0x219f8ba12d6dd5c4076af98d9b608ab10351294d4433fde115fbd23243b48306   |
++-----------------------+------------+----------------------------------------------------------------------+
+
+The ``additional_hash`` and the ``balance_hash`` were calculated in the previous steps and we can now use them in the
+``balance_proof``.
+
+In order to create the ``singature` of the ``LockedTransfer`` we first need to pack the ``balance_proof``:
+
+.. code-block:: 
+
+    packed_balance_proof = pack(balance_proof)
+
+    0xe82ae5475589b828d3644e1b56546f93cd27d1a400000000000000000000000000000000000000000000000000000000000001510000000000000000000000000000000000000000000000000000000000000001000000000000000000000000000000000000000000000000000000000000053a1d9479b298eb0a60edaf962f4cf092465456ad7a0265dfe28a0fe3a2a8ecef4e0000000000000000000000000000000000000000000000000000000000000001219f8ba12d6dd5c4076af98d9b608ab10351294d4433fde115fbd23243b48306
+
+5. Signature
+^^^^^^^^^^^^
+
+After getting the packed form of the ``balance_proof`` we have to sign it in order to generate the message signature.
+
+.. code-block:: 
+
+    signature = eth_sign(privkey=private_key, data=packed_balance_proof)
+
+    0x33b336f151f9790f40287655bd412a043be83a03d0136ef5e002229dd04d5b4c2b505b65911251b2a2eb428403de394064bdae0cd8d4a3bb47a10b1a0d924b921c
+
+Preconditions for LockedTransfer
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+For a ``LockedTransfer`` to be considered valid there are the following conditions. The message will be rejected otherwise:
+
+- (PC1) :term:`chain id`, :term:`token network` address, and :term:`channel identifier` refer to a existing channel
+- (PC2) The corresponding channel is in the open state
+- (PC3) :term:`nonce` is increased by ``1`` with respect to the previous message in that direction, e.g.
+:term:`balance proof` (IS THIS CORRECT?)
+- (PC4) :term:`locksroot` must be equal to the hash of a new list of all currently pending locks, always the latest one
+appended at last position
+- (PC5) :term:`locked amount` must increase by exactly ``lock.amount`` [#PC5]_
+- (PC6) The lock's amount (WHAT IS THAT?) must be smaller than the current :term:`capacity` [#PC6]_
+- (PC7) The lock expiration must be greater than the current block number (IS THAT A DATA FIELD? WHY NOT LINKED?)
```

Comment:
So after reading this comment and checking, I realized that my data are kind of wrong since I put `1` in expiration.  The value of `expiration` is the number of the block where the `Lock` expires. So it makes sense that if the current block, for example, is `10395313` the expiration cannot be at the same block but has to be in a future block. Does that clarify the question?